lib/server/proxyClientRequestToApi.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var Backbone = require("../application/Backbone");
+var Configure = require("../application/Configure");
+
+function proxyClientRequestToApi(apiHost){
+    return function(req,response,next){
+        if (!!req.headers['brisket-client']){
+            Configure.backbone();
+            Backbone.ajax( apiHost + req.path)
+            .then(function(data) {
```

Comment:
I have performance concerns with using Backbone ajax as the proxy middleware. I haven't benchmarked it myself, but the provided proxy should add almost no overhead in my opinion. I would argue to go as close to native as possible. I would like to get some feedback from a couple of @wawjr3d @leochen1216 @ggentzke @sylnp0201 @davydog187 on this.